src/components/NewGoalCriteria/NewGoalCriteria.tsx

```diff
@@ -189,14 +392,18 @@ export const GoalCriteriaPreview: React.FC<GoalCriteriaPreviewProps> = ({ achiev
                     </Text>
                     {nullable(status === 'success', () => (
                         <CriteriaList
+                            className={classes.NewGoalCriteriaTable}
+                            goalId={goalId}
                             list={data?.map((criteria) => {
                                 if (criteria.criteriaGoal) {
                                     return {
```

Comment:
is it the same?

Reply:
Yep, i don't want make component interface based on prisma typings